addons/mail/static/src/new/discuss/channel_selector.js

```diff
@@ -140,6 +141,12 @@ export class ChannelSelector extends Component {
         this.inputRef.el.focus();
     }
 
+    get inputPlaceholder() {
+        return this.state.selectedPartners.length > 0
+            ? _t("Press Enter to start")
+            : _t(this.props.category.addTitle);
```

Comment:
no `_t()` wrapping for non-plain strings. And the content is already `_t()` so can simply use it here.